srec_info/main.cc

```diff
@@ -17,10 +17,12 @@
 //      <http://www.gnu.org/licenses/>.
 //
 
+#include <cstdio>
+#include <cstdint>
+#include <cstdlib>
 #include <iomanip>
 #include <iostream>
-#include <cstdlib>
-#include <cstdio>
+#include <limits>
```

Comment:
Useless include. I wanted to use `numeric_limits` but it didn't worked for `uint32_t`.
The limit is now hardcoded as double.